### DISCUSSION TITLE
APT weather Satellite

### DISCUSSION THREAD

**Discussion by STU8897** [AlexandreRouma/SDRPlusPlus#1113] [Ideas]
Hi, I have Sdr Plus Plus on Android, can you add an APT decoder in the next update? Thank You!

**Comment by f4bpp** [comment]
There is no need to add APT decoder : we only need that Rigctl server works as it should. I've tried to connect SDR++ with GPredict to control the frequency for Doppler correction but actually this feature does not work. An APT image transmission can reach more than 15 minutes and without Doppler correction it is impossible to get a good picture.  For APT decoding you also need an audio virtual cable software to redirect the sound output from SDR++ to the best APT decoder : WXtoIMG. It is very simple to setup. After that, your reception complete APT receiver station looks like this : the satellite tracker  control SDR++  frequency and WXtoIMG get the audio from SDR++ for decoding the pictures.

Actually, SDR++ rigctl module is still in development and all the features we need to interact with other software and hardware are unavailable. Like me, wait (hope) and see. APT decoding with Doppler correction is possible with GPredict , WXtoIMG and GQRX,on Linux (and all SDR software on windows) but on Linux, GQRX is much less ergonomic and practical to use.
